tests/conftest.py

```diff
@@ -92,7 +92,7 @@ def geocode_slc_params():
     test_params.output_hdf5_path = f'{output_path}/{output_file_name}'
 
     # path to groups and datasets in output HDF5
-    test_params.grid_group_path = '/science/SENTINEL1/CSLC/grids'
+    test_params.grid_group_path = '/data'
```

Comment:
Let's add the following import immediately following [this](https://github.com/opera-adt/COMPASS/blob/cc0fcd5ea659606fa2f5a601002b2ba2dd2c86d4/tests/conftest.py#L10):
```
from compass.utils.h5_helpers import DATA_PATH
```
Then the following change can be made here to utilize common paths from `h5_helpers.py`:
```suggestion
    test_params.grid_group_path = DATA_PATH
```